packages/apollos-data-prayer/src/prayer-requests/resolver.js

```diff
@@ -80,22 +80,14 @@ export default {
     id: ({ id }, args, context, { parentType }) =>
       createGlobalId(id, parentType.name),
     campus: ({ campusId }, args, { dataSources }) =>
-      (typeof campusId === 'number' &&
-        dataSources.Campus.getFromId(campusId)) ||
-      null,
-    categoryId: ({ categoryId }) =>
-      (typeof categoryId === 'number' && categoryId) || 0,
-    flagCount: ({ flagCount }) =>
-      (typeof flagCount === 'number' && flagCount) || 0,
-    prayerCount: ({ prayerCount }) =>
-      (typeof prayerCount === 'number' && prayerCount) || 0,
-    requestedByPersonAliasId: ({ requestedByPersonAliasId }) =>
```

Comment:
Is this no longer being used?

Reply:
I put the ones back that matter. I took out too many by mistake.